### DISCUSSION TITLE
Invite code

### DISCUSSION THREAD

**Discussion by longduydao99** [neondatabase/neon#1914] [Q&A] [answered]
Can i get an invite code to register an account, pls?

**Comment by kelvich** [comment, marked as answer]
Yes, please send an email to beta@neon.tech and we'll follow up with an invite.

**Reply by longduydao99** [reply]
I did, but haven't received reply TT

**Reply by hairarrow** [reply]
I'm on that same boat

**Reply by natkli** [reply]
Same here TT
